foodsaving/groups/stats.py

```diff
@@ -44,11 +44,11 @@ def group_summary_email(group, recipient_count):
     }])
 
 
-def periodic_task(name):
+def periodic_task(name, extra_fields=dict):
```

Comment:
That's not how python default values work. extra_field will be a callable, not an actual dict.

The standard way out of this is to set `None` and set the actual default in the function: https://reinout.vanrees.org/weblog/2012/04/18/default-parameters.html (solution is at the bottom)